controllers/likes.js

```diff
@@ -0,0 +1,198 @@
+import models from '../models/index';
+
+const { article, likes } = models;
+
+
+/**
+ * @description Likes and Favorites Controller class
+ */
+class Likes {
+  /**
+   * Adds two numbers together.
+   * @param {Object} req .
+   * @param {Object} res The User Object.
+   * @returns {Object}  Response object having message and status for liking artincle
+   */
+  static async likeArticle(req, res) {
+    try {
+      const { slug } = req.params;
+      const { user } = req;
+
+      const checkArticle = await article.findOne({ where: { slug, status: 'Published' } });
```

Comment:
After a discussion with @aaronsekisambu and @niyobobo we decide to use numbers instead of `Published ` and `Draft` so  when rebasing accept those changes